src/components/SliderButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { sliderImage } from "./SliderImageWallet";
+
+type SliderImageWalletProps = {
+  sliderValue: number;
+  handleEventChange: React.ChangeEventHandler<HTMLInputElement>;
+};
+
+export default function WalletImageSlider({
+  sliderValue,
+  handleEventChange,
+}: SliderImageWalletProps) {
+  let textValue: string;
+  if (sliderValue == 11) {
+    textValue = "Plus Cash";
+  } else if (sliderValue == 1) {
```

Comment:
These numbers can actually lose context over time. It might seem apparent right now, but in 1 month time, it might take overhead to understand why these are used. Extract them to a variable with more context

Reply:
Added variables above to give more context.